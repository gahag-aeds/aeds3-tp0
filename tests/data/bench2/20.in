20 10
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 
Sub 4 9
Max 9 14
Add 7 12
Max 10 15
Sub 4 9
Max 7 12
Add 9 14
Sum 10 15
Add 9 14
Max 12 17
