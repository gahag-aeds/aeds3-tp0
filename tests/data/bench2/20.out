70
53
87
261
66
